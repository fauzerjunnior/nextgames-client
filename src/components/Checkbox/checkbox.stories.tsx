import { Meta } from '@storybook/react';
import Checkbox from './checkbox.component';

export default {
  title: 'Checkbox',
  component: Checkbox,
} as Meta;

export const Default = () => <Checkbox />;
