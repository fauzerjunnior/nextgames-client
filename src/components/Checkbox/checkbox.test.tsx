import { render, screen } from '@testing-library/react';
import Checkbox from './checkbox.component';

describe('Checkbox', () => {
  it('should render the Checkbox', () => {
    const { container } = render(<Checkbox />);

    expect(
      screen.getByRole('heading', { name: /Checkbox/i })
    ).toBeInTheDocument();
    expect(container.firstChild).toMatchSnapshot();
  });
});
