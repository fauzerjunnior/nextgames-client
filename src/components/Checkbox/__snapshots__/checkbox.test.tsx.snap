// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checkbox should render the Checkbox 1`] = `
<main
  class=""
>
  <h1>
    Checkbox
  </h1>
</main>
`;
